src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Unknown" xml:space="preserve">
     <value>&lt;Unknown&gt;</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Encountered some issues while loading the project. this will prevent us from providing some features such as full solution analysis for this project and any projects that depend on this project.</value>
```

Comment:
Proper capitalization, please. Also note I think we're well past our localization deadline at this point.